decidim-meetings/app/cells/decidim/meetings/meeting_m/show.erb

```diff
@@ -0,0 +1,21 @@
+<div class="column">
+  <article class="card card--meeting">
+    <div class="card__content">
+      <%= cell("decidim/meetings/meeting_m", model).header %>
```

Comment:
`<%= render :header %>`, I think

Reply:
I've tried it, it also works, do you prefer `<%= render :header %>`